agent/src/main/java/org/openremote/agent/protocol/http/HTTPAgent.java

```diff
@@ -21,6 +21,7 @@
 
 import org.openremote.model.asset.agent.Agent;
 import org.openremote.model.asset.agent.AgentDescriptor;
+import org.openremote.model.http.HTTPMethod;
```

Comment:
?

Reply:
Result of moving HTTPMethod enum from Agent to Model module.